code/modules/jobs/job_types/roguetown/nobility/lady.dm

```diff
@@ -56,21 +56,64 @@
 		beltr = /obj/item/storage/belt/rogue/pouch/coins/rich
 		backr = /obj/item/storage/backpack/rogue/satchel
 		id = /obj/item/clothing/ring/silver
+
+/datum/job/roguetown/lady/after_spawn(mob/living/L, mob/M, latejoin = TRUE)
+	. = ..()
+	if(ishuman(L))
+		var/mob/living/carbon/human/H = L
+		H.advsetup = 1
+		H.invisibility = INVISIBILITY_MAXIMUM
+		H.become_blind("advsetup")
+
+/datum/advclass/lady/traditional
+	name = "Traditional Consort"
+	tutorial = "Trained since you were young to serve in the traditional manner, you serve the Duchy with your knowledge of how to manipulate the economy, politics, and how to stitch something back together."
+	outfit = /datum/outfit/job/roguetown/lady/traditional
+
+	category_tags = list(CTAG_LADY)
+
+/datum/outfit/job/roguetown/lady/traditional/pre_equip(mob/living/carbon/human/H)
 	if(H.mind)
-		H.mind.adjust_skillrank(/datum/skill/misc/stealing, 4, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/music, 4, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/sneaking, 3, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/combat/knives, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/sewing, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/athletics, 4, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/reading, 6, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/medicine, 4, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/craft/cooking, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/riding, 3, TRUE)
+		H.change_stat("intelligence", 3)
+		H.change_stat("perception", 2)
+		H.change_stat("speed", 2)
+		H.change_stat("fortune", 5)
+	ADD_TRAIT(H, TRAIT_SEEPRICES, TRAIT_GENERIC)
+	ADD_TRAIT(H, TRAIT_GOODLOVER, TRAIT_GENERIC)
+
+/datum/advclass/lady/warrior
+	name = "Warrior Consort"
+	tutorial = "Picked for your noticeable skill with the blade, you serve as your spouse, or future spouse, most close guard, protecting them from danger."
+	outfit = /datum/outfit/job/roguetown/lady/warrior
+
+	category_tags = list(CTAG_LADY)
+
+/datum/outfit/job/roguetown/lady/warrior/pre_equip(mob/living/carbon/human/H)
+	if(H.mind)
+		H.mind.adjust_skillrank(/datum/skill/combat/polearms, 4, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/maces, 4, TRUE)
```

Comment:
4 maces/polearms/swords on a Consort? You're better than a Guard, to tag along the plate armor. It gets better maces and polearms than the Duke... and on par maces/polearms with the Guard Captain. Is this not a bit overpowered... along with 2 strength?